doc_source/update-github-repo.md

```diff
@@ -32,10 +32,9 @@ Properties:
   RepositoryDescription: String
   PersonalAccessToken: String
   IsPrivate: Boolean
-  RepositoryOwner: String
```

Comment:
is it intentional we are removing `RepositoryOwner` ? @KishorFlux7

Reply:
Yes, GitHub does not use that anymore. Anyone who creates a repo gets a contributor access and that is all.